sample/simple-videoplayer-channel/source/CraigVenter.srt

```diff
@@ -0,0 +1,1331 @@
+1
+00:00:17,000 --> 00:00:19,000
+You know, I've talked about some of these projects before,
```

Comment:
This file is not needed for the big bunny video right?

Reply:
Yeah, let's remove it.